### PR TITLE
Add gyazo

### PR DESCRIPTION
**Pull request by SrKomodo**
Followed the gidelines, tested with gulp and it builds correctly

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -2402,6 +2402,15 @@
             "gumtree.com"
         ]
     },
+
```
Comment (Nightfirecat) [inline sites.json]: Please remove the trailing space on this line.

File: sites.json [hunk 2]
```diff
@@ -2402,6 +2402,15 @@
             "gumtree.com"
         ]
     },
+    
+    {
+        "name": "Gyazo",
+        "url": "https://gyazo.com/settings",
```
Comment (Nightfirecat) [inline sites.json]: Is it clear on this page how to delete the account? If not, some notes should be added. (If it is, this is fine; I just don't personally know myself)

**Comment by SrKomodo** [comment]
I have fixed the problems you described

**Comment by Nightfirecat** [comment]
Thanks for your contribution!

**Comment by SrKomodo** [comment]
Nice! I just found out the old website today while deleting old accounts, and I went to the old repo, then found out about this new one and went here.
The website is really useful, so it's nice to now it's being worked on

**Pull request merged**